### PR TITLE
Sixth requirement

### PR DESCRIPTION
**Pull request by dareddov**
The admin panel display (under title header) a number which shows the amount of points of interests in the system of "health" type, chosen by women aged between 20 and 30, and the names of PoIs start with "cosm".

### CODE REVIEW

File: app/models/interest.rb [hunk 1]
```diff
@@ -4,4 +4,7 @@ class Interest < ApplicationRecord
   belongs_to :user
 
   validates :name, :interest_type, presence: true
+
+  scope :health_type, -> { where(interest_type: :health) }
+  scope :specific_scope, -> { health_type.joins(:user).where(user_id: User.young_women.select(:id)).where('name like ?', 'cosm%') }
```
Comment (Exelord) [inline app/models/interest.rb]: OK :) ... Let's see how you can improve it ;)
Tip: Lot of unnecessary methods

File: app/models/user.rb [hunk 2]
```diff
@@ -12,6 +12,8 @@ class User < ApplicationRecord
 
   validates :gender, :age, presence: true
 
+  scope :young_women, -> { where('age between ? and ? and gender = ?', 20, 30, 1) }
```
Comment (Exelord) [inline app/models/user.rb]: '1'? I know that this is women but someone else, no :) Could you use enum value here?

**Review by Exelord — CHANGES_REQUESTED** [review]

**Comment by Exelord** [comment]
I reviewed all PR's. Now I wait for tests for all points (all behaviors) and rubocop :)

**Comment by dareddov** [comment]
@Exelord I improve scopes, look for it now. Thanks. :)

**Pull request merged**